src/server/playwright.ts

```diff
@@ -54,6 +58,18 @@ export class Playwright extends SdkObject {
     this.android = new Android(new AdbBackend(), this.options);
     this.selectors = serverSelectors;
   }
+
+  _enablePortForwarding() {
+    this._portForwardingServer.start();
+  }
+
+  _disablePortForwarding() {
+    this._portForwardingServer.stop();
+  }
+
+  _setForwardedPorts(ports: number[]) {
```

Comment:
Who calls this? Should this be combined with "enable"?

Reply:
IMO we need it separately. Since for `launchServer` we first start the socks server, then wait for incoming connections and then setting the forwardedPorts from the client side.